content/zh/docs/tasks/run-application/horizontal-pod-autoscale.md

```diff
@@ -0,0 +1,554 @@
+---
+reviewers:
+- fgrzadkowski
+- jszczepkowski
+- directxman12
+title: Pod 水平自动伸缩
+feature:
+  title: Horizontal scaling
+  description: >
+    Scale your application up and down with a simple command, with a UI, or automatically based on CPU usage.
```

Comment:
I recommend localizing this sentence too.

Reply:
OK. Thanks.

Do you know how these sections works?